logging/logrus/server_interceptors_test.go

```diff
@@ -94,7 +95,7 @@ func (s *logrusServerSuite) TestPingError_WithCustomLevels() {
 		{
 			code:  codes.Unauthenticated,
 			level: logrus.ErrorLevel,
-			msg:   "Unauthenticated is overwritten to ErrorLevel with customCodeToLevel override, which probably didn't work",
+			msg:   "Unauthenticated is overwritten to ErrorLevel with customClientCodeToLevel override, which probably didn't work",
```

Comment:
Should this read `customCodeToLevel` (as it did before) since this is a server interceptor test?